docs/source/pythonapi/examples/mdgxs-part-i.ipynb

```diff
@@ -797,6 +796,14 @@
    },
    "outputs": [
     {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/lib/python3.6/site-packages/numpy/lib/shape_base.py:873: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future\n",
```

Comment:
Turns out my machine has just updated to the point where this is now an error.  Fixed.
- [x]